elyra/contents/handlers.py

```diff
@@ -38,28 +38,22 @@ async def get(self, path=''):
         path = path or ''
         path = url_unescape(path)
 
-        type = self.get_query_argument('type', default=None)
-        if type not in {None, 'file', 'notebook'}:
-            raise web.HTTPError(400, u'Type %r is invalid' % type)
-
         self.log.debug("Parsing file: %s", path)
 
         try:
             model = await self.operation_parser(path)
         except FileNotFoundError as fnfe:
             model = dict(title=str(fnfe))
             self.set_status(404)
+            raise web.HTTPError(404, str(fnfe)) from fnfe
         except IsADirectoryError as iade:
             model = dict(title=str(iade))
             self.set_status(400)
-        except ValueError as ve:
-            model = dict(title=str(ve))
-            self.set_status(400)
-        except Exception as e:
-            model = dict(title=f'Could not parse {path}: {str(e)}')
+            raise web.HTTPError(400, str(iade)) from iade
+        except Exception:
+            model = {"env_list": {}, "inputs": {}, "outputs": {}}
             self.set_status(200)
         else:
-            model = await self.operation_parser(path)
             self.set_status(200)
```

Comment:
nit:  I would just use a single `set_status(200)` here w/o the `else:` since real exceptions are raised and we document why others are tolerated.